react/features/recording/components/LiveStream/index.js

```diff
@@ -1,3 +1,4 @@
 // @flow
 
 export * from './_';
+export * from './functions';
```

Comment:
Don't export anything from a feature's index.js, just use the full path to /functions when importing from another feature.